app/controllers/articles.server.controller.js

```diff
@@ -12,8 +12,17 @@ var mongoose = require('mongoose'),
  * Create a article
  */
 exports.create = function(req, res) {
+
+	// Filters out control fields
+	delete req.body.updatedAt;
+	delete req.body.updatedBy;
+	delete req.body.createdAt;
+	delete req.body.createdBy;
+
 	var article = new Article(req.body);
-	article.user = req.user;
+
+	article.createdBy = req.user;
```

Comment:
I'm afraid I don't understand. 
You mean, to state the values of updatedBy, updatedAt in the articleByID method, before setting the req.article?
I think that could have some problems, because, as I understand it, the intention of the code is to give the middleware a pristine copy of the article, as it is in the db, so that the middleware can do anything with it (like update it or just use it to get its info for other calculations). If we update the fields on loading, then the middleware won't have a pristine copy of the data in db and could not do some operations like, for example, checking if has not been modified.
Did I understand you correctly?

What I tried to do is to only discard dangerous or control fields from the request body, when they are used to update the db object in an automated way (like the _extend(article, req.body)) as that could have some unintended effects or security implications. Other than that, I don't modify the request payload at all.